docs/en/latest/admin-api.md

```diff
@@ -1154,8 +1154,8 @@ SSL resource request address: /apisix/admin/ssls/{id}
 
 | Parameter    | Required | Type                     | Description                                                                                                    | Example                                          |
 | ------------ | -------- | ------------------------ | -------------------------------------------------------------------------------------------------------------- | ------------------------------------------------ |
-| cert         | True     | Certificate              | HTTPS certificate.                                                                                             |                                                  |
-| key          | True     | Private key              | HTTPS private key.                                                                                             |                                                  |
+| cert         | True     | Certificate              | HTTPS certificate. This field supports saving the value in Secret Manager using the [APISIX Secret](../terminology/secret.md) resource.                                                                                             |                                                  |
+| key          | True     | Private key              | HTTPS private key. This field supports saving the value in Secret Manager using the [APISIX Secret](../terminology/secret.md) resource.                                                                                             |                                                  |
```

Comment:
Does it support `environment variables`, i see you only check the prefix `$secret://`?